### PR TITLE
Do not use nfsserver alias anymore

### PR DESCRIPTION
**Pull request by dgdavid**
From nfs-utils-2.3.3 on, it is needed to use the proper service name for nfs-server since the nfsserver alias has been removed (see request).

Related to https://bugzilla.suse.com/show_bug.cgi?id=1116779

### CODE REVIEW

**Comment by dgdavid** [comment]
Thank you for your comment @lslezak 

What  I did was fix/update the whole YARD documentation for this file. Does it look good for you?

**Comment by jreidinger** [comment]
@dgdavid you can try to run `rake check:doc` and see output.

**Comment by lslezak** [comment]
Yes, fix that as well please.

File: .travis.yml [hunk 1]
```diff
@@ -9,3 +9,4 @@ script:
   # the "yast-travis-ruby" script is included in the base yastdevel/ruby image
   # see https://github.com/yast/docker-yast-ruby/blob/master/yast-travis-ruby
   - docker run -it -e TRAVIS=1 -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" yast-nfs-server-image yast-travis-ruby
```
Comment (lslezak) [inline .travis.yml]: The `yast-travis-ruby` also runs `yardoc` by default. If you run it later again then it makes sense to disable that automatic call, that will save some time at Travis.

Use `yast-travis-ruby -x yardoc` option here, see e.g. https://github.com/yast/yast-packager/commit/465cdbce77c24f008b9e36653abf4dea86da6410

Comment (dgdavid) [reply .travis.yml]: Oh! Good to know. Thank you so much!

**Comment by lslezak** [comment]
@dgdavid check if your machine has a correct time, the last commit has date `git show -s --format=%ci` => `2018-11-23 11:12:58 +0000`.

But the current time actually is `LC_ALL=C date --utc` => `Fri Nov 23 11:04:14 UTC 2018`. So the commit date is in the future.

YaST uses the commit date for all files in the created tarball (to have reproducible builds), see https://github.com/openSUSE/packaging_rake_tasks/blob/master/lib/tasks/tarball.rake#L114

**Pull request merged**